### ISSUE TITLE
Slashes in path to bootstrap.php

### ISSUE THREAD

**Issue by SendiMyrkr** [nette-intellij/intellij-nette-tester#37]
Is possible to make something with that? When i work in Windows i've something like "require_once __DIR__ . '/..\..\..\..\bootstrap.php';" in code. Is possible to force change slashes to forward one even when i use Windows?

**Comment by CZechBoY** [comment]
yes, it is annoying when I upload it to Unix server and it complains :-/
Windows understand unix slashes (**/**), so it would be nice to force unix style.

**Comment by mesour** [comment]
@SendiMyrkr you can use PHP constant `DIRECTORY_SEPARATOR` in your code. https://stackoverflow.com/questions/26881333/when-to-use-directory-separator-in-php-code/26881417

**Comment by SendiMyrkr** [comment]
It's a some time, but if i remember well, than problem was in generated path which was generated depending by used OS. Thats a problem when someone works on windows and has linux server.

**Comment by mesour** [comment]
Aha, If it is generated in the plugin, I will look at it and I will try to fix it.
